dev/release/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# This Dockerfile allows to build the package-registry and packages together.
```

Comment:
It's cool that you touched this.

I see a couple of issue with this file. Can we split this Dockerfile into multiple stages (https://docs.docker.com/develop/develop-images/multistage-build/)? There is no need to store source code in the image.

Reply:
I was not even aware multi stage builds exists. I need to look more into this.

Reply:
I had a look at the multi stage builds and this will be exactly what we need. I assume this will also make it obsolete to clean up images from dependencies which were installed through apt-get or similar.

As I'm still iterating on this for now I would like to keep it all together and do the multi stage in a second step if that is ok for you. In any case, it will need quite a bit of optimisations.